crates/ruff_linter/src/rules/flake8_use_pathlib/rules/dotless_pathlib_with_suffix.rs

```diff
@@ -103,13 +107,12 @@ fn is_path_with_suffix_call(semantic: &SemanticModel, func: &Expr) -> bool {
     typing::is_pathlib_path(binding, semantic)
 }
 
-fn add_leading_dot_fix(string: &ExprStringLiteral) -> Option<Fix> {
-    let first_part = string.value.iter().next()?;
-
-    let opener_length = first_part.flags.opener_len();
-    let after_leading_quote = first_part.start().checked_add(opener_length)?;
-
+fn add_leading_dot_fix(string: &ast::StringLiteral) -> Fix {
+    let opener_length = string.flags.opener_len();
+    let after_leading_quote = string
+        .start()
+        .checked_add(opener_length)
```

Comment:
I don't think we need the check here actually. We know it's less than u32 max because there's at least a `")` following after ;)

Reply:
Gah, I forgot you could use `+` between two `TextSize` instances 🤦

Reply:
#14816